### ISSUE TITLE
Determine whether a Punkt model is available without loading it

### ISSUE THREAD

**Issue by dourouc05** [nltk/nltk#3127]
Hi,

In some code, I'd like to check whether a Punkt model is available or not without loading the file (that is, without using `sent_tokenize` with dummy text and the language). The way to do it is not documented at all. 

I dug into `nltk.data.load` and the whole module to come up with a solution. It's really different from what I had to do with stopwords, where the solution is quite easy and searchable (`ntlk.corpus.stopwords.fileids()`). The final solution I found is `nltk.data.find(f'tokenizers/punkt/{language}.pickle')`, which is super brittle and hard to find. 

Would it be possible to have an easier way to determine whether a Punkt model is available for a given language?

**Comment by wittyicon29** [comment]
Certainly! NLTK doesn't provide a straightforward function specifically designed to check the availability of a Punkt model for a particular language. However, you can create a custom function to check the presence of the Punkt tokenizer model file for a given language using the `find` function from `nltk.data`.

For example:

```python
import nltk

def is_punkt_model_available(language):
    model_path = nltk.data.find(f'tokenizers/punkt/{language}.pickle')
    return model_path is not None

# Usage example:
language_to_check = 'english'  # Replace with the language you want to check
if is_punkt_model_available(language_to_check):
    print(f"A Punkt model for '{language_to_check}' is available.")
else:
    print(f"No Punkt model found for '{language_to_check}'.")
```

This function, `is_punkt_model_available`, checks for the existence of the Punkt tokenizer model file for a specified language. It returns `True` if the model file is found and `False` otherwise.

Since this method relies on the internal structure of NLTK's data storage, so it might be considered somewhat brittle, as you mentioned.
